internal/nhooyr.io/websocket/write.go

```diff
@@ -236,6 +236,13 @@ func (mw *msgWriter) close() {
 
 func (c *Conn) writeControl(ctx context.Context, opcode opcode, p []byte) error {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*5)
+	switch opcode {
```

Comment:
introduce this `switch` in case other calls to `writeControl` depend on `context.WithTimeout(ctx, time.Second*5)`. Otherwise I would remove `ctx, cancel := context.WithTimeout(ctx, time.Second*5)` and just use the ctx passed in

Reply:
I'll look into simplifying this by just using the context passed in